mystery_o_matic/clues.py

```
@@ -361,6 +447,30 @@ def string_spanish(self):
 
         return s
 
+    def string_russian(self):
+        r = randint(0, 2)
```

Ruff (0.15.4)

[error] Standard pseudo-random generators are not suitable for cryptographic purposes

(S311)